Library/src/Json/JsonSerializer.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2023 Magnopus LLC
```

Comment:
I was asking myself the question because I have a new file as well - are we supposed to put 2024 there?

Reply:
Good question. I will find out

Reply:
@MAG-AdamThorn Do you have any information on this?